packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -118,9 +121,16 @@ import { type PropType, computed, ref, onBeforeMount } from 'vue'
 import { KButton } from '@kong/kongponents'
 import { BookIcon, AddIcon } from '@kong/icons'
 import composables from '../../composables'
-import type { EmptyStateFeature } from 'src/types/entity-empty-state'
+import { type EmptyStateFeature, type AppearanceTypes, Appearances } from '../../types/entity-empty-state'
```

Comment:
Please follow the existing import format

Reply:
@afrzhussain I misunderstood this comment originally as well.

You were correct to change `src/` -> `../../`. @adamdehaven is referring to having a single import with `type` and non-type usage.

We are trying to move towards to separate imports (I forget this all the time, so apologies if you copied this usage from me).

```
import type { EmptyStateFeature, AppearanceTypes } from '../../types/entity-empty-state'
import { Appearances } from '../../types/entity-empty-state'
```

Reply:
i've already made these changes